sunpy/timeseries/timeseriesbase.py

```diff
@@ -68,25 +68,23 @@ class GenericTimeSeries:
     """
     A generic time series object.
 
+    Parameters
+    ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
+    meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`, optional
+        The metadata giving details about the time series data/instrument.
+    **kwargs : dict
+        Keyword arguments are not used.
+
     Attributes
     ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
     meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`
         The metadata giving details about the time series data/instrument.
-    data : `~pandas.DataFrame`
-        An pandas DataFrame prepresenting one or more fields as a function of time.
-
-    Parameters
-    ----------------
-    filename: `str` or File
```

Comment:
I have no idea what all these parameters are doing here in the docstring, but they don't seem to be used anywhere in the actual code...